src/passes/Outlining.cpp

```diff
@@ -366,19 +407,36 @@ struct Outlining : public Pass {
         // sequence relative to its function is better for outlining because we
         // walk functions.
         auto [relativeIdx, existingFunc] = stringify.makeRelative(seqIdx);
+#if OUTLINING_DEBUG
+        auto seq = OutliningSequence(
+          relativeIdx,
+          relativeIdx + substring.Length,
+          func,
+          stringify.exprs[seqIdx + substring.Length - 1]->type ==
+            Type::unreachable,
```

Comment:
The duplication here creates risk that these non-trivial arguments get out of sync in the debug and non-debug versions. Can you either create a separate variable for them or restrict the scope of the ifdef?

Reply:
okay i restricted the scope.